Makefile

```diff
@@ -18,16 +18,16 @@ OPT_NVCCFLAGS=--compiler-options "$(CFLAGS) $(OPT_CFLAGS)" --linker-options "$(L
 
 ## Targets for Desktop
 
-plugin.dbg.so: hologram.dbg.o C/source/generateHologram.cu
+plugin.dbg.so: hologram.dbg.o source/generateHologram.cu
```

Comment:
As a side note, the README currently says that the plugin should be built using `hologram.dbg.so`` instead of ``plugin.dbg.so``.